FreeRTOS/FreeRTOS
# Update CORTEX R4 demo to use linked resources #1272

Replace batch script in Cortex R4 demo with linked resources, to support non-Windows dev platforms.

Description
-----------
Two commits:
* Remove the batch script, `FreeRTOS/Demo/CORTEX_R4_RM48_TMS570_CCS5/CreateProjectDirectoryStructure.bat`, and replace it with linked resources
* Ignore the output directories and Eclipse user settings files in this demo

Test Steps
-----------
I tested this by doing a clean build of the demo project with Code Composer Studio 12. It compiles successfully with the symlinks instead of copied files.

I've also tested it on a TMS570-based board and it loaded, ran and debugged successfully.

Checklist:
----------


- [ x ] I have tested my changes. No regression in existing tests.

Related Issue
-----------
#1272 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
